src/Components/3DTiles/3DTilesUtils.js

```diff
@@ -42,14 +42,16 @@ export function getFirstTileIntersection(intersects) {
   let first_inter = null;
   let dist_min = 0;
   for (let inter of intersects) {
-    let geomAttributes = inter.object.geometry.attributes;
-    if (!!geomAttributes && !!geomAttributes._BATCHID) {
-      if (!first_inter) {
-        first_inter = inter;
-        dist_min = inter.distance;
-      } else if (inter.distance < dist_min) {
-        first_inter = inter;
-        dist_min = inter.distance;
+    if(inter.object.visible){
+      let geomAttributes = inter.object.geometry.attributes;
+      if (!!geomAttributes && !!geomAttributes._BATCHID) {
```

Comment:
Why double `!` ?

Reply:
Ok I didn't know, then it doesn't matter, it will disappear with the PR of formatting style.